### ISSUE TITLE
Blueprint section vs. Cheat Sheet

### ISSUE THREAD

**Issue by texnixe** [k-next/getkirby.com#75]
@bastianallgeier Durch die CheatSheet Doku, die du jetzt hinzugefügt hast, haben wir den Kram jetzt an zwei Stellen dokumentiert.

Ich würde daher vorschlagen, den Guide an diesen Stellen wieder auszumisten und stattdessen auf die jeweiligen Cheat Sheet Abschnitte verlinken. Was meinst du?

**Comment by bastianallgeier** [comment]
Ja, das hab ich teilweise schon gemacht. Ich bin jetzt aber am überlegen was der bessere Ort ist. Sollte der Guide auf's Cheatsheet verlinken oder sollten die Inhalte eigentlich im Guide sein und wir verlinken nur vom Cheatsheet drauf. Irgendwie hab ich das Gefühl, dass es das Cheatsheet nur die Übersicht sein sollte und die Inhalte im Guide eigentlich besser aufgehoben sind. Was meinst du?

**Comment by texnixe** [comment]
Es wäre aus meiner Sicht ganz gut, den Guide von den ganzen Options zu bereinigen, da er im jetzigen Stadium dadurch viel zu unübersichtlich ist. Also im Guide nur das Grundsätzliche, im Cheat Sheet die Detais. Dann haben wir nicht alles doppelt und außerdem können User, die schon mit den Grundlagen vertraut sind, dann im Cheat Sheet die Details schnell nachschauen.

Alternativ bräuchten wir Snippets, die wir in beide Teile einfügen, dann würde ich die Optionen aber in einem Accordion verstecken.
